index.html

```diff
@@ -9,17 +9,16 @@
     <!-- Bootstrap and supporting libraries -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-
     <link rel="stylesheet" href="new-pipeline/style/dashboards.css">
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
 </head>
 <body>
     <div class="container-fluid">
-        <header>
+        <header class="portal-header-bar">
             <form class="navbar-form inline pull-right">
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn" title="Report bugs in Telemetry dashboards">Report bug</a>
             </form>
-            <h2>Telemetry portal</h2>
+            <h2>Telemetry portal</h2><br>
```

Comment:
What purpose is this `br` serving here? Could it be replaced with CSS?

Reply:
The line break was to increase the spacing between the content and heading, as it was looking a bit squashed

Reply:
Removed <br>, it works without that!